Tasks/InstalliOSProvisioningProfile/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "vsts-tasks-xcode",
+  "version": "1.0.0",
+  "description": "VSTS Xcode Task",
+  "main": "xcode.js",
```

Comment:
The first few lines of package.json should be changed from Xcode specifics to InstalliOSProvisioningProfile.